extensions/amp-story/1.0/amp-story-page-attachment.css

```diff
@@ -125,3 +125,87 @@ amp-story[desktop] .i-amphtml-story-page-attachment-remote  {
   opacity: 1;
  }
 }
+
+/** Page attachment form submissions state styles. */
+
+.i-amphtml-story-page-attachment-form-submission-status {
+  align-items: center !important;
+  background: #FFFFFF !important;
+  border: 1px solid rgba(0, 0, 0, 0.3) !important;
+  border-radius: 12px !important;
+  box-sizing: border-box !important;
+  display: flex !important;
+  font-family: 'Roboto', sans-serif !important;
+  font-size: 14px !important;
+  font-weight: bold !important;
+  height: 54px !important;
+  margin: 16px 0px !important;
+  width: 100% !important;
+  --i-amphtml-story-page-attachment-form-submission-status-icon-height: 26px !important;
+}
+
+[theme="dark"] .i-amphtml-story-page-attachment-form-submission-status {
+  background: #202125 !important;
+  border: 1px solid rgba(255, 255, 255, 0.2) !important;
+}
+
+[submit-success] .i-amphtml-story-page-attachment-form-submission-status {
+  color: #0F8514 !important;
+}
+
+[submit-error] .i-amphtml-story-page-attachment-form-submission-status {
+  color: #DE3333 !important;
+}
+
+@keyframes submission-status-fade-in {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+[submitting] .i-amphtml-story-page-attachment-form-submission-status,
+.i-amphtml-story-page-attachment-form-submission-status div {
+  /* The entire [submitting] div will fade in along with its border, after which only the contents
+  of the [submit-success] and [submit-error] divs will fade in. */
+  animation: submission-status-fade-in 0.3s !important;
```

Comment:
Plz use decelerated easing to show the div, and accelerated easing to hide it (if needed)
https://material.io/design/motion/speed.html#easing

.2s to show, .15s to hide should also work better given how small the element is

Reply:
Updated, thanks for the link!